### PR TITLE
fix: homepage styles according to ui revision

### PR DESCRIPTION
**Pull request by rafaelramalho19**
- Fix homepage styles according to UI revision from https://www.figma.com/file/826Zurcs6kgWSzQ35it966/web3.storage?node-id=2062%3A565

### CODE REVIEW

File: packages/website/components/navbar.js [hunk 1]
```diff
@@ -24,31 +24,31 @@ export default function Navbar({ bgColor = '', user, isLoadingUser }) {
...
             <>
               <Link href="/files">
-                <a className="text-w3storage-purple font-bold no-underline hover:underline align-middle mr-6 md:mr-12">
+                <a className="text-sm text-w3storage-purple font-bold no-underline hover:underline align-middle mr-6 md:mr-12">
                   Files
                 </a>
               </Link>
               <Link href="/account">
-                <a className="text-w3storage-purple font-bold no-underline hover:underline align-middle mr-6 md:mr-12">
+                <a className="text-sm text-w3storage-purple font-bold no-underline hover:underline align-middle mr-6 md:mr-12">
```
Comment (alanshaw) [inline packages/website/components/navbar.js:51]: Since these are smaller now can we please add some padding to make the click area bigger?

**Review by alanshaw — CHANGES_REQUESTED** [review]
Changes LGTM, just a request for a bit better accessibility 🙏

File: packages/website/components/navbar.js (hunk 1 above)
Comment (rafaelramalho19) [reply packages/website/components/navbar.js]: Good catch, done :)

**Review by alanshaw — APPROVED** [review]
LGTM. I added a tweak to ensure the spacing was consistent between the menu items and the login/logout button.

**Pull request merged**